outputs/policyreport.go

```diff
@@ -0,0 +1,333 @@
+package outputs
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/falcosecurity/falcosidekick/types"
+	"github.com/google/uuid"
+	"github.com/kubernetes-sigs/wg-policy-prototypes/policy-report/kube-bench-adapter/pkg/apis/wgpolicyk8s.io/v1alpha2"
+	wgpolicy "github.com/kubernetes-sigs/wg-policy-prototypes/policy-report/kube-bench-adapter/pkg/apis/wgpolicyk8s.io/v1alpha2"
+	crdClient "github.com/kubernetes-sigs/wg-policy-prototypes/policy-report/kube-bench-adapter/pkg/generated/v1alpha2/clientset/versioned"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/retry"
+)
+
+//for n namespace specific reports with their count
+type pr struct {
+	report *wgpolicy.PolicyReport
+	count  int
+}
+
+const (
+	clusterPolicyReportBaseName = "falco-cluster-policy-report-"
+	policyReportSource          = "Falco"
+	highpriority                = "high"
+	lowpriority                 = "low"
+	mediumpriority              = "medium"
+)
+
+var (
+	minimumPriority string
+	severity        string
+	//count for cluster policy report
+	repcount int
+	//slice of policyreports and their counts(type pr)
+	polreports = make(map[string]*pr)
+	//for cluster policyreport
+	report *wgpolicy.ClusterPolicyReport = &wgpolicy.ClusterPolicyReport{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: clusterPolicyReportBaseName,
+		},
+		Summary: v1alpha2.PolicyReportSummary{
+			Fail: 0,
+			Warn: 0,
+		},
+	}
+	stringToIntForPriority = map[string]int{
```

Comment:
This should be in priority.go, as it could help other outputs

Reply:
I just thought of utilising the already existing functions in priority.go. The changes can be seen in the latest commit.

Reply:
I could add this in priority.go, if you would suggest. 
Earlier I was using this extra map which wasn't really needed.